packages/app/background-jobs-common/src/background-job-processor/processors/AbstractBackgroundJobProcessor.spec.ts

```diff
@@ -529,4 +529,28 @@ describe('AbstractBackgroundJobProcessor', () => {
       })
     })
   })
+
+  describe('getJobCount', () => {
+    it('works', async () => {
```

Comment:
is this a test for lazy init specifically?

Reply:
well, not only lazy init, it is also testing that counting works

Reply:
You are right, I did it quickly so didn't think too much on name ... sorry, will improve it now